docs/user_guide.md

```diff
@@ -0,0 +1,46 @@
+# Using S2I images
+
+When you have tried to use OpenShift builder images and found out that your application requires yet other actions for building/running then you are trying to leverage user scripts from `.s2i/bin`. To make your path smooth here we have collected some approaches and not always obvious receipts.
```

Comment:
>I thought we had higher aspirations for this document, that it would be a general usage guide, but i guess until we have that we can keep it simple.

A journey of a thousand miles begins with a single step (c)

> that said, this paragraph still needs work. How about: "S2I builder images normally include assemble and run scripts, but the default behavior of those scripts may not be suitable for all users. This document will cover a few approaches for customizing the behavior of an S2I builder that includes default assemble/run scripts."

Ok, I updated this paragraph but I removed last occurrence of "assemble/run" to reduce repetition.